get-involved.html

```diff
@@ -31,7 +31,7 @@ <h2>Get Involved</h2>
 					<div class="col-md-8 col-md-offset-2 animate-box text-center">
 						<h2 class="intro-heading">Participate</h2>
 						<p>CivicTechWR brings together people from different sectors and industries to actively solve issues facing our local community. We welcome people with all types and levels of skills. Even though "tech" is in the name, everyone is welcome and everyone has a skill or experience that is valuable. You don’t need to know anything about the local government either, even though “civic” is in the name.</p>
-						<p>If you have a project idea, announcement, or event, you’re welcome to pitch at any of our hacknights, or join our <a href="https://civictechwrslack.herokuapp.com/">slack team</a> to talk to the group.</p>
+						<p>If you have a project idea, announcement, or event, you’re welcome to pitch at any of our hacknights, or join our <a href="https://civictechwr.slack.com/">slack team</a> to talk to the group.</p>
```

Comment:
Are `Slack` settings open enough so that anyone with the link to the group can join? Or one needs an invitation to join? If invitation is required then this change might not make sense. Maybe we need to ask people to email someone to get an invite link or post the invite link here itself?